### PR TITLE
Addition of the attribute name for the subnet in resource_mso_schema_site_vrf_region

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

**Review by xinyuezhao — APPROVED** [review]
LGTM

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

File: mso/resource_mso_schema_site_vrf_region.go [hunk 1]
```diff
@@ -108,6 +108,11 @@ func resourceMSOSchemaSiteVrfRegion() *schema.Resource {
 										Required:     true,
 										ValidateFunc: validation.StringLenBetween(1, 1000),
 									},
+									"name": &schema.Schema{
+										Type:         schema.TypeString,
+										Optional:     true,
```
[the remark this answers is not in the document]
Comment (shrsr) [reply mso/resource_mso_schema_site_vrf_region.go:113]: All the attributes in the datasource file except the ones that are Required should be Computed and that doesn't go vice versa with the resource file.
In this case I have set 'name' to optional in resource in order for it to be consistent with the other attribute 'zone'. Let me know if you want me to change both to optional and computed which is possible.

[the remark this answers is not in the document]
Comment (anvitha-jain) [reply mso/resource_mso_schema_site_vrf_region.go:113]: Adding "computed : true" when "optional: true" is recommended (Unless it disrupts the working part of the existing code.)

File: website/docs/d/schema_site_vrf_region.html.markdown [hunk 2]
```diff
@@ -39,7 +39,8 @@ data "mso_schema_site_vrf_region" "vrfRegion" {
 * `cidr.primary` - (Optional) primary flag to set above ip as primary for cidr. Only one ip must be set as primary.
```
Comment (anvitha-jain) [inline website/docs/d/schema_site_vrf_region.html.markdown:39]: ip -> IP

File: website/docs/r/schema_site_vrf_region.html.markdown [hunk 3]
```diff
@@ -52,7 +53,8 @@ resource "mso_schema_site_vrf_region" "vrfRegion" {
 * `cidr.primary` - (Required) Primary flag to set above ip as primary for cidr. Only one ip must be set as primary.
```
Comment (anvitha-jain) [inline website/docs/r/schema_site_vrf_region.html.markdown:53]: ip -> IP

**Review by anvitha-jain — CHANGES_REQUESTED** [review]

**Review by anvitha-jain — APPROVED** [review]
LGTM

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**